airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/scheduling/RouterService.java

```diff
@@ -37,9 +43,16 @@ public String getTaskQueue(final UUID connectionId, final TemporalJobType jobTyp
     return taskQueueMapper.getTaskQueue(geography, jobType);
   }
 
+  // This function is only getting called for discover/check functions. Today (02.07) they are behind
```

Comment:
I don't think I'm following the feature flag strategy here. If a workspace isn't in the `routeTaskQueueForWorkspaceAllowList`, should we instead just avoid calling the `getTaskQueueForWorkspace` method entirely?

Reply:
ideally yes, but this logic actually has been on prod since this morning already so the flag is built to support https://github.com/airbytehq/airbyte-platform-internal/pull/4369 - I think essentially they should reach the same result?

Reply:
Clarified via Slack, I was misunderstanding the current state! This makes sense.